data/npc/lib/npcsystem/npcsystem.lua

```diff
@@ -51,7 +51,7 @@ if NpcSystem == nil then
 	-- Gets an npcparameter with the specified key. Returns nil if no such parameter is found.
 	function NpcSystem.getParameter(key)
 		local ret = getNpcParameter(tostring(key))
-		if (type(ret) == 'number' and ret == 0) or ret == nil then
+		if (type(ret) == 'number' and ret == 0) or not ret then
```

Comment:
`ret == nil` is redundant. It can be dropped completely since `return ret` will still return nil in the case that `ret` is `nil`.

From what I noticed `getNpcParameter` returns either a string or nil, so `type(ret)` will never be `'number'`.